apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -669,9 +678,15 @@ export class NotificationService {
 
   private async sendCourtDateNotifications(
     theCase: Case,
-    user: User,
+    user?: User,
     eventOnly?: boolean,
   ): Promise<SendNotificationResponse> {
+    if (!user) {
```

Comment:
Is this needed? Why did we change the user from being a required param?

Reply:
The notification service can now be called by initiated by the message handler where there is no user involved.